Update and rename add-network.md to wallet-api.md (#425)
0xPolygon/polygon-docs: docs/tools/wallets/venly/wallet-api.md

!!! caution "Content disclaimer"
    Please view the third-party content disclaimer [here](https://github.com/0xPolygon/polygon-docs/blob/main/CONTENT_DISCLAIMER.md).

# Wallet API

The Wallet API allows developers to interact with blockchain networks and offer wallet functionality to their users without having to build everything from scratch. This can include features like account creation, transaction management, balance inquiries, and more.

- Welcome your users with custom wallet branding. You can customize the user interface to your taste.
- You are completely in charge of the wallet user experience to optimize your user conversion. Get total freedom on UX and asset management with our Wallet API.

> - You and your users have complete control over digital assets, without any third-party interference. Securely manage wallets with complete autonomy and privacy.

Reviewer: ```suggestion
- You and your users have complete control over digital assets without any third-party interference. Securely manage wallets with complete autonomy and privacy.
```

> - In case of lost login details, you and your users can recover access to wallets with a security code or biometric verification.

Reviewer: ```suggestion
- In the event of loss of login credentials, you and your users can recover access to wallets with a security code or biometric verification.
```

![Wallet-API to create Polygon wallets](../../../img/tools/wallet/venly/07.png)

## Key features

| Features | Description |
| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Wallet management           | Developers can use the API to create, manage, and secure wallets for their users.                                                                                           |
| Transaction services        | The API can enable the initiation and monitoring of blockchain transactions.                                                                                                |
| Token support               | It may allow the handling of various tokens and assets on supported blockchain networks.                                                                                    |

> | Blockchain interactions     | Developers can integrate functionalities like reading data from the blockchain or writing data to it, more commonly known as creating and interacting with Smart Contracts. |

Reviewer: ```suggestion
| Blockchain interactions     | Developers can integrate functionalities like reading data from the blockchain or writing data to it, along with creating and interacting with smart contracts. |
```
